app/models/piece.rb

```diff
@@ -93,4 +93,20 @@ def remains_on_board?(to_x:, to_y:)
       false
     end
   end
+
+  def successful_capture(to_x:, to_y:)
+    return false if obstructed_diagonally?(to_x:, to_y:)
+    return false if obstructed_vertically?(to_x:, to_y:)
+    return false if obstructed_horizontally?(to_x:, to_y:)
+    current_color = current_piece.color
+    target_piece = Piece.where(x_position: to_x, y_position: to_y, active: true, color: !current_color)  # Add requirement that target MUST be opponent's color
```

Comment:
Yes, that `if ... elsif` would work, esp. if we wanted to display the `piece is friend` or `piece is foe` somewhere. But if we don't want to, would it be more concise to simply edit line #103 to read: ```if target_piece.exists? && target_piece.color != color```